src/main/java/uk/gov/hmcts/reform/bailcaseapi/domain/handlers/postsubmit/BailApplicationSubmittedConfirmation.java

```diff
@@ -26,7 +26,7 @@ public PostSubmitCallbackResponse handle(Callback<BailCase> callback) {
 
         postSubmitResponse.setConfirmationBody(
             "### What happens next\n\n"
-            + "All parties will be notified that the application has been submitted."
+            + "All sendDirectionList will be notified that the application has been submitted."
```

Comment:
The word "parties" got accidentally swapped for "sendDirectionList".

Reply:
I have changed sendDirectionList back to parties